src/platform/forms-system/src/js/fields/ArrayField.jsx

```diff
@@ -348,7 +349,10 @@ export default class ArrayField extends React.Component {
                   <button
                     type="button"
                     className="usa-button-secondary edit vads-u-flex--auto"
-                    aria-label={`Edit ${itemName}`}
+                    aria-label={`
+                      Edit ${itemName} ${formData[index][supplementalLabel] ??
```

Comment:
I love the nullish coalescing operator 😎 

I do think we want to keep the itemName only option, since we still need an aria label in the event a uiSchema doesn't contain a supplementalLabel

Reply:
Oh wait. I misread this lol. This works just fine 🎉